### PR TITLE
fix: update warp route guides to use --symbol instead of --warp

### PR DESCRIPTION
**Pull request by ltyu**
The goal is to reduce friction of having users find the warp route artifacts in their file system. Instead, we will have users just select their token by symbol, which fetches the artifacts in the background

### CODE REVIEW

File: docs/guides/remove-trusted-relayer.mdx [hunk 1]
```diff
@@ -4,26 +4,28 @@
...
+- The warp route token symbol used in `hyperlane warp deploy`.
+    - The token symbol can be found onchain, or in your local Hyperlane Registry. By default, it is written to `$HOME/.hyperlane/deployments/warp_routes/<tokenSymbol>`.
 - Access to the private key that currently owns the warp route.
 
 :::info
 If you followed the [Deploy a Warp Route](https://docs.hyperlane.xyz/docs/guides/deploy-warp-route) guide, you most likely have deployed a warp route with a trusted relayer set to a signer address.
 :::
 
-To confirm using the Hyperlane CLI, locate your warp route under the `/deployments/warp_routes` directory, and execute the following with `address` set to the `addressOrDenom` of your warp route:
+To confirm using the Hyperlane CLI, locate your token symbol and the chain it is deployed on:
```
Comment (jmrossy) [inline docs/guides/remove-trusted-relayer.mdx:21]: Pedantic grammar nit: the chain on which it is deployed

File: docs/guides/remove-trusted-relayer.mdx [hunk 2]
```diff
@@ -4,26 +4,28 @@
...
 
 ```bash
-hyperlane warp read --address <addressOrDenom> --chain yourchain
+hyperlane warp read --symbol <tokenSymbol> --chain <yourchain>
 ```
 
+You should be able to select the warp route that you wish to remove the trusted relayer for.
```
Comment (jmrossy) [inline docs/guides/remove-trusted-relayer.mdx:27]:
```suggestion
You can then choose the warp route for which you're removing the trusted relayer.
```

File: docs/guides/transfer-warp-route-ownership.mdx [hunk 3]
```diff
@@ -7,40 +7,35 @@ One of the quickest way to transfer a warp route ownership is by using the [Hype
...
 
-To confirm using the Hyperlane CLI, locate your warp route under the `/deployments/warp_routes` directory, and execute the following with `address` set to the `addressOrDenom` of your warp route:
+To confirm using the Hyperlane CLI, run the folllowing command with your token symbol and the chain it is deployed on:
 
 ```bash
-hyperlane warp read --address <addressOrDenom> --chain yourchain
+hyperlane warp read --symbol <tokenSymbol> --chain <yourchain>
 ```
 
+You should be able to select the warp route that you wish to update the ownership for.
```
Comment (jmrossy) [inline docs/guides/transfer-warp-route-ownership.mdx:25]: If all this is duplicated, should it be in a shared place, like `docs/partials` or `src/components`?

**Review by jmrossy — COMMENTED** [review]
Looks good but I wonder if we should take this opportunity to DRY up the duplicated text

File: docs/guides/transfer-warp-route-ownership.mdx (hunk 3 above)
Comment (ltyu) [reply docs/guides/transfer-warp-route-ownership.mdx]: Moved most duplicates to partials

File: docs/guides/remove-trusted-relayer.mdx (hunk 1 above)
Comment (ltyu) [reply docs/guides/remove-trusted-relayer.mdx:19]: fixed

**Review by jmrossy — APPROVED** [review]

**Pull request merged**